cartographer/mapping/pose_graph.h

```diff
@@ -26,8 +26,8 @@
 #include "cartographer/common/lua_parameter_dictionary.h"
 #include "cartographer/mapping/id.h"
 #include "cartographer/mapping/pose_graph_trimmer.h"
+#include "cartographer/mapping/proto/pose_graph.pb.h"
```

Comment:
This change will also be necessary in cartographer_ros/cartographer_ros/assets_writer_main.cc and perhaps others on the ros side.

Reply:
Of course. But it makes no sense to send the follow up PR until this is in. And it only needs changes to `assets_writer_main.cc`, no others.